pr_agent/git_providers/bitbucket_provider.py

```diff
@@ -26,6 +26,13 @@ def __init__(self, pr_url: Optional[str] = None, incremental: Optional[bool] = F
         if pr_url:
             self.set_pr(pr_url)
 
+    def get_repo_settings(self):
+        try:
+            contents = self.repo_obj.get_contents(".pr_agent.toml", ref=self.pr.head.sha).decoded_content
+            return contents
+        except Exception:
+            return ""
```

Comment:
**Suggestion:** Consider adding error handling for when the '.pr_agent.toml' file is not found in the repository. Currently, the 'get_repo_settings' function will return an empty string if an exception occurs, which could be misleading.
```suggestion
    def get_repo_settings(self):
        try:
            contents = self.repo_obj.get_contents(".pr_agent.toml", ref=self.pr.head.sha).decoded_content
            return contents
        except Exception as e:
            if isinstance(e, FileNotFoundError):
                logging.warning('.pr_agent.toml file not found in repository.')
            else:
                logging.error('Failed to get repository settings.', exc_info=e)
            return None
```